bugbug/bug_features.py

```diff
@@ -85,15 +85,20 @@ def __call__(self, bug):
 
 class whiteboard(object):
     def __call__(self, bug):
-        ret = []
 
-        # TODO: Add any [XXX:YYY] that appears in the whiteboard as [XXX: only
+        paren_splits = bug['whiteboard'].lower().split('[')
+        splits = []
+        for paren_split in paren_splits:
+            if ']' in paren_split:
+                paren_split = paren_split.split(']')
```

Comment:
Is this needed? Can't we just remove the ']' characters?

Reply:
@marco-c  In case like `[whiteboard1] whiteboard`, just removing the `]` will result in both the words getting joined. Hence, it's better to split at ']'.